packages/design/jobberStyle.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const fs = require("fs");
+// eslint-disable-next-line import/no-internal-modules
+const customPropertiesObject = require("./foundation.js");
+
+const customProperties = customPropertiesObject.customProperties;
+const allKeys = Object.keys(customProperties);
+const resolvedCssVars = allKeys.reduce((acc, key) => {
+  const newKey = key.replace("--", "");
+  acc[newKey] = jobberStyle(key);
+  return acc;
+}, {});
+
+const jsonContent =
+  "export const JobberStyle = " + JSON.stringify(resolvedCssVars, undefined, 2);
+
+fs.writeFile("./foundation.js", jsonContent, "utf8", function(err) {
+  if (err) {
+    console.log("An error occured while writing JSON object to File.");
+    return console.log(err);
+  }
+  console.log("JSON file has been saved.");
+});
+
+function jobberStyle(styling) {
+  const varRegExp = /var\((.*)\)/;
+  const calcRegExp = /calc\((.*)\)/;
```

Comment:
Can we use a post css plugin to handle calc for us ahead of this script?

Reply:
the postcss plugin wasn't handling calcs that have css vars.
`'--space-smallest': 'calc(var(--base-unit)/8)',` doesn't get evaluated and stays as is
if the expression was `'--space-smallest': 'calc(8/8)',` it gets resolved to `1`